front/src/components/Foods.vue

```diff
@@ -0,0 +1,63 @@
+<template lang="pug">
+  v-layer
+    v-group(
+      v-for="food in foods"
+      :key="food.id"
+    )
+      k-image(:config="buildFoodConfig(food)")
+</template>
+
+<script>
+import KImage from './KImage'
+
+export default {
+  name: 'Foods',
+  components: {
+    KImage,
+  },
+  props: {
+    foods: {
+      type: Array,
+      default () {
+        return []
+      },
+    },
+    gridSpacing: {
+      type: Number,
+      default () {
+        return 25
```

Comment:
Is the 25 supposed to be hard-coded here? If so maybe it needs a comment. Otherwise I'm not sure why this is 25.

Reply:
It is an arbitrary default value.